app/javascript/dashboard/components/layout/sidebarComponents/AccountSelector.vue

```diff
@@ -58,5 +66,43 @@ export default {
       globalConfig: 'globalConfig/get',
     }),
   },
+  methods: {
+    async onChangeAccount(accountId) {
+      const accountUrl = `/app/accounts/${accountId}/dashboard`;
+      window.location.href = accountUrl;
+      await this.$store.dispatch('setActiveAccount', {
+        accountId: accountId,
+      });
+    },
+  },
```

Comment:
Would this work? As the call would happen only after the redirect is complete. 

I guess we should call this whenever we are loading the app. Every time we load the app, we can check if the account is the same as the previously active account. If not, call the API to set it as active.

Reply:
Makes sense